tests/core/contracts/test_contract_call_interface.py

```diff
@@ -16,6 +16,7 @@
 pytestmark = pytest.mark.filterwarnings("ignore:implicit cast from 'char *'")
 
 
+# TODO use transact to deploy
```

Comment:
Yeah, this TODO can be removed. The deprecation warnings that show up in the tests should be sufficient.

Reply:
fixed